elastic/kibana
# [Synthetics] add alerting config migration

## Summary

Resolves https://github.com/elastic/kibana/issues/154380

Adds a migration for including alert configurations for documents < 8.6.0 where the alert config was not included.

### Testing

1. Check out 8.5.0
2. Create a new oblt-cli cluster, or run `yarn es snapshot` to connect to ES
3. Log into Kibana on 8.5.0. Create a monitor
4. Check out this branch. 
5. Start Kibana and log into Kibana. Click on the monitor edit page for the monitor you created. Inspect the network request for the monitor to confirm `alert.status.enable: true` has been added to the monitor attributes
